control/subscription_group.go

```diff
@@ -383,7 +377,7 @@ func (s *subscriptionGroup) subscribePlugins(id string,
 	for i, sub := range plugins {
 		plg, err := s.pluginManager.get(key(sub))
 		if err != nil {
-			serrs = append(serrs, serror.New(err))
+			serrs = append(serrs, pluginNotFoundError(sub))
```

Comment:
After looking at the code, I would think the error handling would be better suited in `pluginManager.get` instead of `loadedPlugins.get`. At that point, `loadedPlugins.get` could possibly be simplified to return the loadedPlugin or nil, and let `pluginManager.get` decide what error to return: pluginNotFound or invalid key when no plugin is returned.